config/jobs/periodic/docker-in-docker/periodic-dind-build.yaml

```diff
@@ -0,0 +1,213 @@
+            fi
+
+            ls /workspace 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            
+            if [[ -f env.list ]]
+            then
+              # check there are 3 env variables in env.list
+              cat env.list 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              if grep -Fq "DOCKER_VERS" env.list && grep -Fq "CONTAINERD_VERS" env.list && grep -Fq "PACKAGING_REF" env.list
+              then 
+                echo "DOCKER_VERS, CONTAINERD_VERS, PACKAGING_REF are in env.list" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              else 
+                echo "DOCKER_VERS, CONTAINERD_VERS and/or PACKAGING_REF are not in env.list" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+                exit 1
+              fi
+              set -o allexport
+              source env.list
+            else
+              echo "There is no env.list" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            fi
+            
+            # generate the env-distrib.list
+            echo "*** * env-distrib.list * ***" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            mkdir docker-ce-packaging
+            pushd docker-ce-packaging
+            git init
+            git remote add origin  https://github.com/docker/docker-ce-packaging.git
+            git fetch --depth 1 origin ${PACKAGING_REF}
+            git checkout FETCH_HEAD
+
+            make REF=${DOCKER_VERS} checkout
+            popd
+
+            # get the packages list in the env_distrib.list
+            echo DEBS=\"`cd docker-ce-packaging/deb && ls -1d debian-* ubuntu-*`\" > env-distrib.list
+            echo RPMS=\"`cd docker-ce-packaging/rpm && ls -1d centos-* fedora-*`\" >> env-distrib.list
+
+            rm -rf docker-ce-packaging
+
+            # check the env.list (versions of docker-ce, containerd and list of packages)
+            if [[ -f env-distrib.list ]]
+            # if there is env.list and env-distrib.list
+            then
+              cat env-distrib.list 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              # check if there are two variables in env-distrib.list
+              if grep -Fq "DEBS" env-distrib.list && grep -Fq "RPMS" env-distrib.list
+              then 
+                echo "DEBS and RPMS are in env-distrib.list" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              else 
+                echo "DEBS and/or RPMS are not in env-distrib.list" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+                exit 1
+              fi
+              source env-distrib.list
+            else
+              echo "There is no env-distrib.list" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              exit 1
+            fi
+
+            # build docker_ce and containerd
+            echo "*** ** BUILD ** ***" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            CONT_NAME=docker-build
+            docker run --env DOCKER_SECRET_AUTH --env PATH_SCRIPTS --env PATH_LOG_PROWJOB -d -v /workspace:/workspace --privileged --name $CONT_NAME ${PATH_IMAGE_BUILD} ${PATH_SCRIPTS}/build.sh 
+            docker logs -f $CONT_NAME
```

Comment:
Please see if checking of `docker logs -f $CONT_NAME` is needed?

Reply:
Yes, sorry, I used that for the tests, I forgot to remove it